hudi-common/src/test/java/org/apache/hudi/common/fs/inline/TestInLineFileSystemHFileInLining.java

```diff
@@ -40,18 +42,18 @@
 import java.util.Set;
 import java.util.UUID;
 
-import static org.apache.hudi.common.fs.inline.FileSystemTestUtils.FILE_SCHEME;
-import static org.apache.hudi.common.fs.inline.FileSystemTestUtils.RANDOM;
-import static org.apache.hudi.common.fs.inline.FileSystemTestUtils.getPhantomFile;
-import static org.apache.hudi.common.fs.inline.FileSystemTestUtils.getRandomOuterInMemPath;
+import static org.apache.hudi.common.testutils.FileSystemTestUtils.FILE_SCHEME;
+import static org.apache.hudi.common.testutils.FileSystemTestUtils.RANDOM;
+import static org.apache.hudi.common.testutils.FileSystemTestUtils.getPhantomFile;
+import static org.apache.hudi.common.testutils.FileSystemTestUtils.getRandomOuterInMemPath;
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 
 /**
  * Tests {@link InLineFileSystem} to inline HFile.
  */
-public class TestHFileInLining {
+public class TestInLineFileSystemHFileInLining {
```

Comment:
Can we rename to `TestInLineFileSystem`?

Reply:
@yanghua `TestInLineFileSystem` already exists in the same package. This is testing `TestInLineFileSystem` for a group of cases for HFile inlining.

Reply:
OK, sounds good.